pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -117,6 +117,13 @@ def __getstate__(self):
         state["predict_progress_bar"] = None
         return state
 
+    @staticmethod
+    def _resolve_refresh_rate(refresh_rate: Optional[int]) -> int:
```

Comment:
```suggestion
    def _resolve_refresh_rate(refresh_rate: int) -> int:
```

Reply:
None could be passed.

Reply:
This is inconsistent with the type hint for refresh rate in the constructor, which expects an int